### PR TITLE
Feature/reg postcode component

### PR DESCRIPTION
**Pull request by hdfedscottquested**

### CODE REVIEW

File: components/atoms/icon/__snapshots__/Icon.test.js.snap [hunk 1]
```diff
@@ -7,8 +7,9 @@ exports[`Check the icon renders it should match the snapshot 1`] = `
     Object {
       "backgroundColor": "#0085ff",
       "borderColor": undefined,
-      "height": 39,
-      "width": 39,
+      "borderWidth": 3,
+      "height": 60,
+      "width": 60,
```
Comment (hdfeddanjackson) [inline components/atoms/icon/__snapshots__/Icon.test.js.snap:12]: Remove the comma from the end of line 12.

File: components/atoms/identifier/Identifier.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import classNames from 'classnames'
+import PropTypes from 'prop-types'
+import './identifier.scss'
+
+const Identifier = props => {
```
Comment (hdfeddanjackson) [inline components/atoms/identifier/Identifier.js:6]: I don't really see the point of this component as it's just returning a `<span>` element.

File: components/atoms/icon/__snapshots__/Icon.test.js.snap (hunk 1 above)
Comment (hdfedscottquested) [reply components/atoms/icon/__snapshots__/Icon.test.js.snap:12]: Generated content mate

File: components/atoms/identifier/Identifier.js (hunk 2 above)
Comment (hdfedscottquested) [reply components/atoms/identifier/Identifier.js:6]: I know what you mean. It is however a component within the designs, so technically it still needs to be a component. It outputs the box with the car reg or postcode in it.

Comment (hdfeddanjackson) [reply components/atoms/identifier/Identifier.js:6]: I still feel that this component has no real purpose beyond a unique use case. All we're doing here is to render a `<span>` element to fit a specific purpose. This is not really a reusable component, so we should look for a different solution.

Comment (hdfedscottquested) [reply components/atoms/identifier/Identifier.js:6]: What would you suggest in its place?

Comment (hdfeddanjackson) [reply components/atoms/identifier/Identifier.js:6]: What are we trying to accomplish? I just cannot see the point of having a component that simply renders a `<span>` element, to me that's not a worthwhile component.

Comment (hdfedscottquested) [reply components/atoms/identifier/Identifier.js:6]: This is a small component that is used in quite a few places across different molecules/Organisms/Pages so having this as a component made sense to me. Happy to change it if you have a better way

**Comment by hdfedscottquested** [comment]
Closed as moved this into header component

**Pull request closed without merging**